_i18n/es/resources/user-guides/easiest_buy.md

```diff
@@ -1,66 +1,65 @@
+{% include disclaimer.html translated="true" version=page.version %}
+## Cómo adquirir Monero
 
-This is a guide to obtain your own Monero as of 20150919. This is perhaps the easiest way to purchase and hold Monero.
+Esta es una guía para obtener tu propio Monero, a 20150919. Este es quizá el método más fácil de comprar y guardar Monero.
 
-####Step 1: Buy Bitcoin
+####Paso 1: Comprar Bitcoin
 
-There are many ways to buy Bitcoin. Perhaps the easiest way is through circle.com. Once you have purchased some Bitcoin, you are ready to buy some Monero! Buying Bitcoin is straightforward. Please goto circle.com and just follow the instructions there.
+Hay muchas formas de comprar Bitcoin. Quizá la forma más fácil es a través de circle.com. Una vez que hayas comprado algo de Bitcoin, estás listo para comprar Monero! Comprar Bitcoin es sencillo. Ve a circle.com y sigue las instrucciones ahí.
 
-####Step 2: Set up a mymonero.com account
+####Paso 2: Configura una cuenta mymonero.com
 
-MyMonero.com is an online wallet for Monero, maintained by Monero Core Developer Ricardo Spagni (fluffpony). It is the easiest wallet to use. Simply go to MyMonero.com and click on the "Create an Account" button.
+MyMonero.com es un monedero online para Monero, mantenido por el desarrollador central de Monero, Ricardo Spagni (fluffypony). Es el monedero más fácil de usar. Simplemente ve a MyMonero.com y haz clic en el botón Create an Account.
 
 ![image1](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/easiest_way/01.png)
 
-After clicking the button, you will see your private key. This key is what gives you access to your funds. Never share this key with anyone!
+Después de hacer clic, verás tu clave privada. Esta llave es lo que te dará acceso a tus fondos. ḂNunca compartas esta clave con nadie!
 
-### WRITE DOWN THIS KEY IMMEDIATELY!
+### ESCRIBE Y GUARDA ESTA LLAVE INMEDIATAMENTE
 
 ![image2](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/easiest_way/02.png)
 
-Type in your private key in the box below, and click the button.
+Escribe tu clave privada en el espacio mostrado, y haz clic en el botón
 
-On the next page, you will see your address.
+En la siguiente página, podrás ver tu dirección.
 
 ![image3](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/easiest_way/03.png)
 
-Copy your address to the clipboard by highlighting the whole thing and hitting ctrl+c (or edit menu, copy), or clicking the little icon next to your address. Save your address somewhere. This is how others will send Monero to you, and what you will use to deposit Monero into your account!
+Copia tu dirección en el portapapeles seleccionando toda la cadena y presionando Ctrl+C (o en Edit menu, Copy), o presionando el pequeño icono al lado de tu dirección. Guarda tu dirección en algún lugar. Con ella otros usuarios podrán enviarte Monero, y será lo que utilizarás para depositar Monero en tu cuenta.
```

Comment:
(o en el menu Edit, Copy) o (o en el menu Edit, la palabra Copy)
(o en el menu Editar, Copiar) o (o en el menu Editar, la palabra Copiar)

No se si traducirlo también si el sistema operativo de quien utilice el manual este en español.

Reply:
También tuve esa duda, puede ser y puede que no, además no sé si mymonero . com esté disponible también en español, por ejemplo.

Reply:
Lo pondré como en la primer sugerencia por ahora.

Reply:
Ahí entendi que hace referencia a la barra del menu del sistema operativo, por eso te sugeri cambiarlo incluso mas abajo en otra traducción to lo pones editar y copiar por eso también te lo sugeri.